packages/devtools_app/lib/src/config_specific/import_export/import_export.dart

```diff
@@ -71,23 +71,20 @@ class ImportController {
     }
     previousImportTime = now;
 
-    final isDevToolsSnapshot =
-        json is Map<String, dynamic> && json[devToolsSnapshotKey] == true;
-    if (!isDevToolsSnapshot) {
+    final devToolsSnapshot = json;
```

Comment:
this change is fine to land externally, but the other changes to remove the null assert operators won't pass on the bots externally. DevTools is built from flutter 2.10.0-0.3.pre externally, which is much older than what is in google3. We can't move to the latest beta because there is a bug in that version of flutter that prevents apps from being able to connect to DevTools. So until then, we'll have to manually remove the `!` operators in google3 only.

Reply:
Oh good to know, I didn't realize that! That explains why this is failing.